### ISSUE TITLE
Unable to download ITV program on El Capitan

### ISSUE THREAD

**Issue by g8kig** [Ascoware/get-iplayer-automator#225]
_What steps will reproduce the problem? Be specific, and provide as much detail as possible._

Attempt to download Endeavour - Sun Feb 10 2019

_What is the expected output? What do you see instead?_

Expected: Download succeeds.
Actual: XC_BAD_INSTRUCTION in _AES.so

_What version of GiA are you using? On what version of OS X?_
GiA Version 1.14.3 (20190121001)
MacOS 10.11.6

_Are you using a web proxy? DNS proxy? VPN? VPS?_
No (in the UK)

*What program are you attempting to download (full title, series and episode)? 

ITV Download (ID=2a1229a0024): Downloading Endeavour - Sun Feb 10

[GiA bug.txt](https://github.com/Ascoware/get-iplayer-automator/files/2866992/GiA.bug.txt)

Succeeds on Mojave using Xeons

**Comment by skovatch** [comment]
Does any ITV show fail or just this one? This is going to be tough to diagnose/fix, as I don't have any systems running 10.11.

**Comment by g8kig** [comment]
I just tried the next episode of Endeavour (2a1228a0025, 17th Feb) the failure is the same just a little later:

Downloading with a build of youtube_dl on a Windows machine also works.
I have tried Coronation Street 1a0694a9697 Mon 18th Feb. This also fails.
As my 10.11 has a Core2Duo and considering the crash, it seems that the component that crashes is built for a later processor. 
[GiA2.txt](https://github.com/Ascoware/get-iplayer-automator/files/2885923/GiA2.txt)
